docs/conf.py

```diff
@@ -72,7 +72,6 @@
 ]
 autoapi_ignore = [
     "*_test.py",
-    "*/package_data/*",
```

Comment:
Is there a reason we don't want to exclude the "packages" under `package_data` from the auto API docs? I think they're just empty `__init__.py` files so that it's all importable.